.github/workflows/release.yaml

```diff
@@ -30,13 +30,13 @@ jobs:
       - name: "Run GoReleaser"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          HOMEBREW_GETENVOY_GITHUB_TOKEN: ${{ secrets.GETENVOY_CI_GITHUB_TOKEN }}
+          HOMEBREW_FUNC-E_GITHUB_TOKEN: ${{ secrets.FUNC-E_CI_GITHUB_TOKEN }}
```

Comment:
I have to generally fix this as we need to redo the homebrew and backfill an alias, but this only happens at release so going to do that after this PR.